eZ/Publish/Core/FieldType/DateAndTime/SearchField.php

```diff
@@ -76,4 +77,32 @@ public function getDefaultSortField()
     {
         return $this->getDefaultMatchField();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFullTextData(Field $field, FieldDefinition $fieldDefinition)
+    {
+        $dateTime = new DateTime();
+        $dateTime->setTimestamp($field->value->data['timestamp']);
+
+        return [
+            // format taken from {@see \eZ\Publish\Core\FieldType\DateAndTime\Value::$stringFormat}
+            $dateTime->format('U'),
```

Comment:
Seconds since the Unix Epoch? Not sure how usable this is for full text search :)
Also, I'm not sure why this was added? Maybe something for a separate PR.